server/handlers/gist.ts

```diff
@@ -0,0 +1,114 @@
+import { Request, Response, NextFunction } from 'express';
+import request from 'request';
+import { name, version } from './../../package.json';
+
+const getGists = (req: Request, res: Response, next: NextFunction) => {
+  console.log(req.user);
+  request.get(
+    `https://api.github.com/users/${req.user.username}/gists`,
+    {
+      json: true,
+      headers: {
```

Comment:
We need to use common authorize token from environment variables for Github's request limit.

Reply:
Oh, I see !!
I will add common authorize token into code and then pull request again.